packages/atomic/src/components/common/facets/category-facet/children-as-tree-container.spec.ts

```diff
@@ -0,0 +1,105 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {html} from 'lit';
+import {describe, it, expect} from 'vitest';
+import {renderCategoryFacetChildrenAsTreeContainer} from './children-as-tree-container';
+
+interface CategoryFacetChildrenAsTreeContainerProps {
```

Comment:
We should import the interface from children-as-tree-container.ts instead or redefining it here